docs/src/main/sphinx/connector/exasol.md

```diff
@@ -0,0 +1,205 @@
+  :header-rows: 1
+
+  * - Exasol database type
+    - Trino type
+    - Notes
+  * - ``BOOLEAN``
+    - ``BOOLEAN``
+    -
+  * - ``DOUBLE PRECISION``
+    - ``REAL``
+    -
+  * - ``DECIMAL(p, s)``
+    - ``DECIMAL(p, s)``
+    -  See :ref:`exasol-number-mapping`
+  * - ``CHAR(n)``
+    - ``CHAR(n)``
+    -
+  * - ``VARCHAR(n)``
+    - ``VARCHAR(n)``
+    -
+  * - ``DATE``
+    - ``DATE``
+    -
+```
+
+No other types are supported.
+
+(exasol-number-mapping)=
+### Mapping numeric types
+
+An Exasol `DECIMAL(p, s)` maps to Trino's `DECIMAL(p, s)` and vice versa
+except in these conditions:
+
+- No precision is specified for the column (example: `DECIMAL` or
+  `DECIMAL(*)`).
+- Scale (`s`) is greater than precision.
+- Precision (`p`) is greater than 36.
+- Scale is negative.
+
+(exasol-character-mapping)=
+### Mapping character types
+
+Trino's `VARCHAR(n)` maps to `VARCHAR(n)` and vice versa if `n` is no greater
+than 2000000. Exasol does not support longer values.
+If no length is specified, the connector uses 2000000.
+
+Trino's `CHAR(n)` maps to `CHAR(n)` and vice versa if `n` is no greater than 2000.
+Exasol does not support longer values.
+
+```{include} jdbc-type-mapping.fragment
+```
+
+(exasol-sql-support)=
+## SQL support
+
+The connector provides {ref}`globally available <sql-globally-available>` and
+{ref}`read operation <sql-read-operations>` statements to access data and
+metadata in the Exasol database.
+
+## Table functions
+
+The connector provides specific {doc}`table functions </functions/table>` to
+access Exasol.
+
+(exasol-query-function)=
+### `query(varchar) -> table`
+
+The `query` function allows you to query the underlying database directly. It
+requires syntax native to Exasol, because the full query is pushed down and
+processed in Exasol. This can be useful for accessing native features which are
+not available in Trino or for improving query performance in situations where
+running a query natively may be faster.
+
+```{include} query-passthrough-warning.fragment
+```
+
+As a simple example, query the `example` catalog and select an entire table::
+
+```sql
+SELECT
+  *
+FROM
+  TABLE(
+    example.system.query(
+      query => 'SELECT
+        *
+      FROM
+        tpch.nation'
+    )
+  );
+```
+
+As a practical example, you can use the
+[WINDOW clause from Exasol](https://docs.exasol.com/db/latest/sql_references/functions/analyticfunctions.htm#AnalyticFunctions):
+
+```sql
+SELECT
+  *
+FROM
+  TABLE(
+    example.system.query(
+      query => 'SELECT
+        id, department, hire_date, starting_salary,
+        AVG(starting_salary) OVER w2 AVG,
+        MIN(starting_salary) OVER w2 MIN_STARTING_SALARY,
+        MAX(starting_salary) OVER (w1 ORDER BY hire_date)
+      FROM employee_table
+      WINDOW w1 as (PARTITION BY department), w2 as (w1 ORDER BY hire_date)
+      ORDER BY department, hire_date'
+    )
+  );
+```
+
+```{include} query-table-function-ordering.fragment
+```
+
+## Performance
```

Comment:
Lets remove from here on down .. we can add in later when we implement a first pushdown or whatever.